sample-queries/sample-queries.json

```diff
@@ -1845,6 +1845,204 @@
       "tip": "This query requires the TI indicator external id. To find the ID, you can run: GET https://graph.microsoft.com/beta/security/tiIndicators?$top=5",
       "skipTest": false
     },
+    {
+      "id": "d4eb9650-719c-4c42-a124-c26549b671e7",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get articles",
+      "requestUrl": "/beta/security/threatIntelligence/articles",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
```

Comment:
Instead of linking to the files in the PR, I'd recommend using the build version: https://review.learn.microsoft.com/en-us/graph/api/security-article-list?view=graph-rest-beta&branch=2023-03-01-threatIntelligence

This will mean that reviewers can actually see the documentation. It will make updating this much easier once we're merged and published, just needing to remove `review.` and remove the `&branch=<branch>`